### PR TITLE
Add rsqrt  to ops API

### LINKED ISSUE #688: Add `rsqrt` to keras_core.ops
Jax API : https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.rsqrt.html
TF API : https://www.tensorflow.org/api_docs/python/tf/math/rsqrt
Torch API : https://pytorch.org/docs/stable/generated/torch.rsqrt.html

Numpy doesn't have it.

If we get this in, [this layer](https://github.com/keras-team/keras-nlp/blob/b04123b369413988c4d6dad1f25652bee29edcdf/keras_nlp/models/t5/t5_layer_norm.py#L36) in KerasNLP will become free of TF and we could use core ops.rsqrt directly !

I'm up for contributing this.

### PR DESCRIPTION
**Pull request by shivance**
Closes #688

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! Please add unit tests for correctness and shape inference (in `keras_core/ops/math_test.py`)

**Comment by shivance** [comment]
> Thanks for the PR! Please add unit tests for correctness and shape inference (in `keras_core/ops/math_test.py`)

Done!

File: keras_core/ops/math.py [hunk 1]
```diff
@@ -450,3 +450,34 @@ def fft2(a):
...
+
+    def call(self, x):
+        x = backend.convert_to_tensor(x)
+        return backend.math.rsqrt(x)
+
+    def compute_output_spec(self, x):
+        return KerasTensor(x.shape, dtype=x.dtype)
+
+
+@keras_core_export(["keras_core.ops.rsqrt", "keras_core.ops.math.rsqrt"])
```
Comment (fchollet) [inline keras_core/ops/math.py:478]: You don't need the `.math` export, there is no such namespace.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**